azure-pipelines.yml

```diff
@@ -50,6 +50,9 @@ jobs:
         node-6:
           node_version: ^6.9.0
           webpack_version: latest
+        webpack-new:
+          node_version: ^10.13.0
+          webpack_version: 5.0.0-alpha.26
```

Comment:
next

Reply:
Sorry , but I just found out this pr is same as https://github.com/webpack/webpack-dev-server/pull/2244, since the issue was not tagged from other pr , I end up making a pr .

I will go ahead and close it :).

Thanks for your time